src/views/RecIM/views/Activity/components/ScheduleList/index.jsx

```diff
@@ -122,16 +165,21 @@ const ScheduleList = ({
           {standardDate(series.StartDate, false)}, or the earliest available day, at{' '}
           {format(Date.parse(series.Schedule.StartTime), 'h:mmaaa')}.{' '}
         </Typography>
+        {(series.Type === 'Round Robin' || series.Type === 'Ladder') &&
+          parameterFields(series.Type, series.TeamStanding.length)}
       </Typography>,
     );
     setOpenAutoSchedulerDisclaimer(true);
     closeMenusAndForms();
   };
 
   const handleConfirmAutoSchedule = () => {
+    let parameterLabel =
+      series.Type === 'Round Robin' ? 'roundRobinMatchCapacity' : 'numberOfLadderMatches';
```

Comment:
The casing of these does not match the `AutoScheduleParameters` type you've declared that they should match.

Reply:
I forgot to save my `.ts` file whoops